spot_rl_experiments/experiments/skill_test/test_dynamic_yaw_plus_mobile_gaze.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+import os.path as osp
+import time
+
+import magnum as mn
+import numpy as np
+import rospy
+from spot_rl.envs.skill_manager import SpotSkillManager
+from spot_rl.utils.utils import ros_topics as rt
+from spot_wrapper.utils import get_angle_between_two_vectors
+
+NUM_REPEAT = 2
+WAYPOINT_TEST = [[5.4, 2.5]] * NUM_REPEAT  # x, y
+
+
+class SpotRosSkillExecutor:
+    """This class reads the ros butter to execute skills"""
```

Comment:
ros buffer?

Reply:
Its Rosparam

Reply:
I meant `butter` is a typo.. can you change it to buffer or param,

Reply:
typo fixed